### DISCUSSION TITLE
Save settings in Docker

### DISCUSSION THREAD

**Discussion by stevenbaert** [open-webui/open-webui#703] [General]
Hi,

Love this project!
Maybe I overlooked the answer but how can I save settings when I run this in Docker?
Thanks!

Steven

**Comment by tjbck** [comment]
Hi! You can provide additional env variables listed here when executing the `docker run` command: https://github.com/ollama-webui/ollama-webui/blob/main/Dockerfile, https://github.com/ollama-webui/ollama-webui/blob/main/backend/config.py

```
ENV OPENAI_API_BASE_URL ""
ENV OPENAI_API_KEY ""
```

**Reply by jasonkrasavage** [reply]
Can you elaborate? I can't seem to figure out for the life of me where or how to modify open webui environment variables.

**Reply by frenzybiscuit** [reply]
> Can you elaborate? I can't seem to figure out for the life of me where or how to modify open webui environment variables.

You set the environmental variables in your docker compose file.
